cve_bin_tool/checkers/bluez.py

```diff
@@ -6,13 +6,34 @@
 
 """
 from . import Checker
+from re import search
 
 
 class BluezChecker(Checker):
-    CONTAINS_PATTERNS = []
+    CONTAINS_PATTERNS = [
+        r"Dialog Semiconductor GmbH",
+        r"Nordic Semiconductor ASA",
+    ]
     FILENAME_PATTERNS = [r"bluetoothctl", r"libbluetooth.so"]
     VERSION_PATTERNS = [
-        r"bluetoothctl: ([5]+\.[0-9]+\.[0-9]+)",
-        r"bluetoothctl: ([5]+\.[0-9]+)",
+        r"^([5]+\.[0-9]+)$",
```

Comment:
This pattern is too generic and will match things that are not bluetoothctl.  Also, is there a reason the other patterns were removed?  In general, we want to add patterns but rarely remove them because old patterns are usually still useful for matching old versions, which is a key feature of this tool.

Reply:
> This pattern is too generic and will match things that are not bluetoothctl.  Also, is there a reason the other patterns were removed?  In general, we want to add patterns but rarely remove them because old patterns are usually still useful for matching old versions, which is a key feature of this tool.

I checked out the version 5.42 for which the checker was written for. I couldn't find the version pattern there. I also checked the older versions. Couldn't find them there as well. That's why I override the get_version() method so this pattern will work only for certain filenames

Reply:
Hrm, if they don't work for anything you coud find, it's entirely possible that this is just a bad checker and we should remove/disable it.  I assumed it worked for some older versions, but that may not be true if we can't get any reasonable tests working.

Reply:
Note: this checker was merged during hacktoberfest, if I recall correctly, so may not have ever had really good tests.